### PR TITLE
BSWH - biospecimen add/update references

### PR DESCRIPTION
**Pull request by Gbarra9**
This pull request is related to: 
- https://github.com/episphere/connect/issues/882

Code changes
- Added BSWH code references 
- Added BSWH collection and site locations:
- 
_BCC- All Saints (FW)
BCC- Plano, BCC
Worth St, BCC- Irving
BCC- HWC
NTX Biorepository_

**Note: Having this pull request reviewable and ready after May release**

### CODE REVIEW

**Review by anthonypetersen — APPROVED** [review]

File: src/fieldToConceptIdMapping.js [hunk 1]
```diff
@@ -262,33 +262,39 @@ export const conceptIds = {
...
+        120264574: 'South Loop',
+        691714762: 'Rice Lake',
+        487512085: 'Wisconsin Rapids',
+        983848564: 'Colby Abbotsford',
+        261931804: 'Minocqua',
+        665277300: 'Merrill',
+        467088902: 'Fargo South University',
+        111111111: 'NIH/NCI',
+        807835037: 'Other',
+        723351427: 'BCC- HWC, BC',
```
Comment (amber-emmes) [inline src/fieldToConceptIdMapping.js:292]: Minor readability concern but were the extra indents here intentional?

File: src/shared.js [hunk 2]
```diff
@@ -1945,6 +1946,7 @@ export const siteFullNames = {
     'KPCO': 'Kaiser Permanente Colorado',
     'HP': 'HealthPartners Research Clinic',
     'HFHS': 'Henry Ford Health System',
+    'BSWH': 'Baylor Scott & White Health',
```
Comment (amber-emmes) [inline src/shared.js:1949]: Same comment as the other PR about BSWH vs baylorSWH

File: src/fieldToConceptIdMapping.js (hunk 1 above)
Comment (Gbarra9) [reply src/fieldToConceptIdMapping.js:292]: I am not sure if this Github displaying this incorrectly. My code has them aligned without the extra indents.

**Review by amber-emmes — APPROVED** [review]
Looks good.

**Pull request merged**